proto/value.proto

```diff
@@ -19,8 +20,6 @@ message Value {
         Document document = 12;
         bytes uuid = 16;
         Zone timezone = 19;
-        DateTime datetime = 20;
-        ZonedDateTime zoned = 21;
```

Comment:
The protobuf Timestamp type is a zoned datetime. Eventually supporting specifically dates and times seems like it could be good, but is there an advantage to have a non-zoned datetime? I just went ahead and cleaned up the protos for those two types we are not using.